tools/build/build.js

```diff
@@ -21,39 +21,84 @@ if (NODE_VERSION < NODE_VERSION_TARGET) {
   process.exit(1);
 }
 
+const STANDARD_BUILD = "Standard Build"
+const TGS_BUILD = "TGS Build"
+const ALL_MAPS_BUILD = "CI All Maps Build"
+const TEST_RUN_BUILD = "CI Integration Tests Build"
+
+let BUILD_MODE = STANDARD_BUILD;
+if(process.env.CBT_BUILD_MODE != undefined){
```

Comment:
```suggestion
if(process.env.CBT_BUILD_MODE !== undefined){
```

Reply:
It doesn't make a difference here, but `==` and `!=` are never correct in the world of JavaScript.